Import error

`var inky = require('react-inky');`

`Error: Cannot find module 'react-inky'
    at Function.Module._resolveFilename (module.js:440:15)
    at Function.Module._load (module.js:388:25)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (D:\Documentos\GitHub\react-ink\index.js:1:74)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)`

--- a/package.json
+++ b/package.json
@@ -2,7 +2,7 @@
   "name": "react-inky",
   "version": "0.1.0",
   "description": "React components for Inky",
-  "main": "index.js",
+  "main": "src/index.js",
   "scripts": {
     "docs:build": "gitbook build",
     "docs:serve": "gitbook serve",

